src/session.ts

```diff
@@ -534,6 +554,11 @@ export class Session extends common.GrpcServiceObject {
   private _getSpanner(): Spanner {
     return this.parent.parent.parent as Spanner;
   }
+
+  private channelId(): number {
+    // TODO: Infer channelId from the actual gRPC channel.
+    return 1;
```

Comment:
The Node.js client does not use a channel pool, meaning that returning 1 for `channelId` seems to be the correct thing to do in all cases. Meaning: We should replace the TODO with something like this:

```suggestion
    // The Node.js client does not use a gRPC channel pool, so this function always returns 1.
    return 1;
```